lsp-mode.el

```diff
@@ -5509,14 +5509,19 @@ If ACTION is not set it will be selected from `lsp-code-actions-at-point'."
         (lsp--info "No formatting changes provided")
       (lsp--apply-text-edits edits))))
 
-(defun lsp-organize-imports ()
-  "Perform the source.organizeImports code action, if available."
-  (interactive)
-  (condition-case nil
-      (lsp-execute-code-action-by-kind "source.organizeImports")
-    (lsp-no-code-actions
-     (when (called-interactively-p 'any)
-       (lsp--info "source.organizeImports action not available")))))
+(defmacro lsp-make-interactive-code-action (func-name code-action-kind)
+  "Define an interactive function FUNC-NAME that attempts to
+execute a CODE-ACTION-KIND action."
+  `(defun ,func-name ()
```

Comment:
I would suggest to automatically append prefix `lsp-` into `func-name`, since its cleaner than looking at two lsp name in same line.
So it will become
`(lsp-make-interactive-code-action organize-imports...)